### PR TITLE
solid tau now uses correct bruggeman coeff

### LINKED ISSUE #1734: solid phase phase tortuosity is calculated with bruggeman coefficient of electrolyte
- PyBaMM version - `$ python -m pip show pybamm`: not sure, since i "git fetch" from github repo to hardrive now and then, last git fetch was on 24th of september 2021.
- Python version - `$ python --version`: 3.9.7

**Describe the bug**
In the file bruggeman-tortuosity a clear distinction between Electrolyte and Electrode is made via an if-elif statement. Yet in both parts of the statement the bruggeman coefficient of the electrolyte phase is used (compare line 41 with line 51).

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to 'pybamm/models/submodels/tortuosity/bruggeman_tortuosity.py'
2. Compare "if" statement starting at line 31 and the "elif" statement starting at line 42. Line 41 and 51 are the same but they shouldn't be. They both calculate tortuosities with their correct respective volume fraction but use the bruggeman coefficient of the electrolyte ("param.b_e_n"). This way the tortuosity of the electrode (line 51) is calculated by putting the active material volume fraction to the power of the bruggeman coefficient of the electrolyte phase. Instead the tortuosity of the electrode should be put to the power of the bruggeman coefficient of the electrode phase.

**Expected behaviour**
I expected the tortuosity of electrode and electrolyte to be scaled differently by their bruggeman coefficients but this is not the case.

**Additional context**
Error can be fixed by 

- replacing "param.b_e_n" in line 47 by "param.b_s_n" and
- replacing "param.b_e_p" in line 51 by "param.b_s_p"

### PR DESCRIPTION
**Pull request by dion-w**
# Description
Solid tortuosity now uses solid bruggeman coefficient of respective electrode and not the bruggeman coefficients of the electrolyte.

Fixes #1734

## Type of change

Please add a line in the relevant section of [CHANGELOG.md](https://github.com/pybamm-team/PyBaMM/blob/develop/CHANGELOG.md) to document the change (include PR #) - note reverse order of PR #s. If necessary, also add to the list of breaking changes.

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (back-end change that speeds up the code)
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Added line to relevant section of changelog  

# Key checklist:

- [x] No style issues: `$ flake8`
- [x] All tests pass: `$ python run-tests.py --unit`
- [x] The documentation builds: `$ cd docs` and then `$ make clean; make html`

You can run all three at once, using `$ python run-tests.py --quick`.

## Further checks:

- [ ] Code is commented, particularly in hard-to-understand areas
- [ ] Tests added that prove fix is effective or that feature works
- not deemed necessary as this only changes two lines of code in the respective directory

### CODE REVIEW

**Comment by dion-w** [comment]
Since this is my first pull request i appreciate feedback for handling this process.

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@
 
 ## Bug fixes
  
+-   solid tortuosity is now correctly calculated with bruggeman coefficient of the respective electrode ([#1773](https://github.com/pybamm-team/PyBaMM/pull/1773))
```
Comment (brosaplanella) [inline CHANGELOG.md:13]:
```suggestion
-   Solid tortuosity is now correctly calculated with Bruggeman coefficient of the respective electrode ([#1773](https://github.com/pybamm-team/PyBaMM/pull/1773))
```

**Review by brosaplanella — COMMENTED** [review]
Looks good, but there is a test related to the sensitivities which is not passing. @martinjrobins is it just a matter of the tolerances or is there a bigger issue?

**Comment by tinosulzer** [comment]
Let's try using even tighter tolerances for solving in the sensitivities test: change these both to 1e-12 https://github.com/pybamm-team/PyBaMM/blob/269629fc83774dbdea2288f466195b8805e72df2/tests/integration/test_models/standard_model_tests.py#L113-L114

Also, could you make a new section "Unreleased" for the changelog? i.e. at the top add
```
# [Unreleased](https://github.com/pybamm-team/PyBaMM/)

## Bug fixes
- Solid tortuosity is now correctly calculated with Bruggeman coefficient of the respective electrode ([#1773](https://github.com/pybamm-team/PyBaMM/pull/1773))
```
Since these changes aren't in the 21.10 release

**Comment by tinosulzer** [comment]
I tested this locally and the sensitivity test passed. Can you make the changelog update, merge develop, and push again?

**Comment by tinosulzer** [comment]
@dion-w any update on this?

**Comment by dion-w** [comment]
Haven't been active here recently. Will hopefully be after a deadline on thursday.

**Review by tinosulzer — APPROVED** [review]
Thanks @dion-w !

**Review by brosaplanella — APPROVED** [review]

**Comment by brosaplanella** [comment]
@all-contributors please add @dion-w for bug reports and code

**Comment by brosaplanella** [comment]
The notebook tests are failing because we need to provide the Bruggeman parameters for the electrode in cell [16] of https://github.com/pybamm-team/PyBaMM/blob/develop/examples/notebooks/parameterization/parameterization.ipynb

**Pull request merged**